zulipterminal/cli/run.py

```diff
@@ -185,11 +185,14 @@ def main(options: Optional[List[str]]=None) -> None:
     except Exception as e:
         if args.debug:
             # A unexpected exception occurred, open the debugger in debug mode
-            import pudb
-            pudb.post_mortem()
 
-        sys.stdout.flush()
-        traceback.print_exc(file=sys.stderr)
```

Comment:
With this change, if debug is *not* enabled then an unexpected exception will cause what appears to be a regular exit to a user, I think - which seems rather undesirable? Could you add something like an `in_color('red', "zulip-term exited unexpectedly")`, perhaps with reference to the final log-file name we decide upon, if one was created?